test/parallel/test-crypto-keygen.js

```diff
@@ -778,6 +787,23 @@ const sec1EncExp = (cipher) => getRegExpForPEM('EC PRIVATE KEY', cipher);
   }));
 }
 
+// Test EdDSA key generation.
+{
+  if (!/^1\.1\.0/.test(process.versions.openssl)) {
```

Comment:
This check looks wrong. I'm pretty sure 0.9.8 doesn't support EdDSA. :-)

(Jokes aside, this check whitelists 1.0.2, which doesn't seem right.)

Reply:
I think we cannot compile / link against 1.0.2 (or 0.9.8) anymore.

Reply:
v8.x bundles 1.0.2 but now that I think of it, that release line is in maintenance mode, won't receive these changes, and therefore doesn't matter. Okay, objection withdrawn.